### DISCUSSION TITLE
Data Removal Request - FaceBook Login

### DISCUSSION THREAD

**Discussion by rwstorer** [umputun/remark42#1652] [Q&A] [answered]
As part of implementing a FaceBook login app, we must provide a privacy policy and a way for users to remove all their data.

The privacy policy is pretty easy (although, I'm certainly open to any suggestions). How would you recommend doing the data removal (preferably in an automated, user clickable, sort of way)?

Thanks!

**Comment by umputun** [comment]
pls, use discussion for questions. I have transferred this for you

**Comment by umputun** [comment, marked as answer]
Regarding privacy policy, you can make whatever policy you like or link to https://remark42.com/privacy/; this worked for me on multiple sites with remark42 installed. Please note that Facebook also asks, "Where can we see the link to the policy," - the answer is "The bottom line of the official site's page https://remark42.com".

For user-data removal, this functionality is already in place. 

![image](https://github.com/umputun/remark42/assets/535880/c20c0050-134a-49a5-a3a9-400c1aaa9623)

![image](https://github.com/umputun/remark42/assets/535880/3d899cb1-4e51-46b0-a0bb-2d229065a2bd)

_@paskal - I think we missed this part in docs unless I'm blind and can't find it_

**Reply by umputun** [reply]
In case you want to implement your custom data removal by combining deleting of your data with remark42 data, take a look at API, it is [well documented](https://remark42.com/docs/contributing/api/)
